### PR TITLE
fix Embedded ObjectMeta in volumegroup crds

### PR DESCRIPTION
**Pull request by matancarmeli7**
Signed-off-by: matancarmeli7 <matan.carmeli7@gmail.com>

### CODE REVIEW

File: volume-group-operator/config/crd/bases/csi.ibm.com_volumegroupclasses.yaml [hunk 1]
```diff
@@ -17,54 +17,54 @@ spec:
     listKind: VolumeGroupClassList
     plural: volumegroupclasses
     shortNames:
-      - vgclass
```
Comment (ArbelNathan) [inline volume-group-operator/config/crd/bases/csi.ibm.com_volumegroupclasses.yaml:20]: please realign, I don't think we need to reformat the indentations every time we will touch this file

File: volume-group-operator/config/crd/bases/csi.ibm.com_volumegroupcontents.yaml [hunk 2]
```diff
@@ -46,10 +46,10 @@ spec:
           description: VolumeGroupContent is the Schema for the volumegroupcontents API
           properties:
             apiVersion:
-              description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+              description: "APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources"
```
Comment (ArbelNathan) [inline volume-group-operator/config/crd/bases/csi.ibm.com_volumegroupcontents.yaml:49]: why the `"`,`'` diff?

Comment (matancarmeli7) [reply volume-group-operator/config/crd/bases/csi.ibm.com_volumegroupcontents.yaml:49]: this is how the Yaml generation does it I think your IDE changed it

File: volume-group-operator/config/crd/bases/csi.ibm.com_volumegroupclasses.yaml (hunk 1 above)
Comment (matancarmeli7) [reply volume-group-operator/config/crd/bases/csi.ibm.com_volumegroupclasses.yaml:20]: this is how the YAML generation does it, I think your IDE changed it at first, like [here](https://github.com/IBM/ibm-block-csi-operator/pull/292/files#diff-e97b4b6c488cb189abdde0029c0a9a8f5745029f104073ac9fe6af2feb1ffbd8)

**Review by ELENAGER — APPROVED** [review]
LGTM

**Review by ArbelNathan — APPROVED** [review]
LGTM

**Pull request merged**